ansible/controller_files/playbook.yml

```diff
@@ -0,0 +1,68 @@
+- name: Deploy ToDo app
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: secret_key
+      private: true
+      prompt: Enter the Flask secret key
+      salt_size: 8
+
+    - name: trello_key
+      private: true
+      prompt: Enter the Trello key
+      salt_size: 8
+    
+    - name: trello_token
+      private: true
+      prompt: Enter the Trello token
+      salt_size: 8   
+
+  tasks:
+    - name: Install yum Packages
+      yum:
+        name:
+          - git
+          - python3
+      become: true
+
+    - name: Install poetry
+      ansible.builtin.shell: curl -sSL https://install.python-poetry.org | python3 -
+      args:
+        creates: ~/.local/bin/poetry
+      become: true
+
+    - name: Create To-Do app directory
+      ansible.builtin.file:
+        path: /opt/todoapp
+        state: directory
+        mode: "777"
+      become: true
+
+    - name: Clone git repo
+      ansible.builtin.git:
+        repo: https://github.com/TomJSoftwire/Devops-Course-Starter
+        dest: /opt/todoapp
+        version: main
+
+    - name: Install dependencies
+      ansible.builtin.shell: source ~/.bash_profile; cd /opt/todoapp; poetry install
```

Comment:
The inbuilt approach for your `cd` step here would be to use the `chdir` parameter. [Here's an example of using that in the Ansible docs](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/shell_module.html#examples)